registry/job.go

```diff
@@ -254,7 +266,7 @@ func dirToHeartbeat(dir *etcd.Node) (heartbeat string) {
 // getUnitFromObject takes a *etcd.Node containing a Unit's jobModel, and
 // instantiates and returns a representative *job.Unit, transitively fetching the
 // associated UnitFile as necessary
```

Comment:
Document what `unitFunc` does. The name `unitHashLookupFunc` might make more sense.